core/src/main/resources/schema.graphqls

```diff
@@ -112,6 +112,8 @@ type Mutation {
 
     addSpriteToGame(gameId: ID!, sprite: NewSprite!): Boolean!
 
+    removeSpriteFromGame(gameId: ID!, sprite: NewSprite!): Boolean!
```

Comment:
Type of argument shouldn't be `NewSprite`. What if a user prepares fake `NewSprite` with nonmatching `name` and `image`?  Only name of the sprite should be sent and the field `image` should be fetched from `Mongo`.

Reply:
Fixed